Makefile

```diff
@@ -223,13 +223,15 @@ test-ci-js: | clear-stalled
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) -p tap --logfile test.tap \
 		--mode=release --flaky-tests=$(FLAKY_TESTS) \
 		$(TEST_CI_ARGS) $(CI_JS_SUITES)
+	! ( ps awwx | grep Release/node | grep -v grep )
```

Comment:
Yes, they're all portable, or at least sufficiently portable that they work on all the non-Windows CI machines.